sdk/cosmosdb/cosmos/src/CosmosClientOptions.ts

```diff
@@ -28,6 +28,15 @@ export interface CosmosClientOptions {
    * Allows users to generating their own auth tokens, potentially using a separate service
    */
   tokenProvider?: TokenProvider;
+  /** AAD token from @azure/identity
+   * Obtain a credential object by creating an @azure/identity credential object
+   * We will then use your credential object and a scope URL (your cosmos db endpoint)
+   * e.g., https://zfoster-rbactest-1.documents.azure.com/.default
```

Comment:
(I am not sure how it works for cosmos..)

Is there no default scope for cosmos?
For example, for libraries such as service-bus, we have the auth_scope set to `https://servicebus.azure.net//.default` in the SDK itself. 

Similarly for all the storage libraries - `https://storage.azure.com/.default`.

Reply:
I tried to use the default scope but the backend seemed to reject it `documents.azure.com/.default`. I'm happy to try again